src/misc/constants.ts

```diff
@@ -205,3 +205,29 @@ export const EVM_CHUNK_SIZES: Record<EVMNetwork, number> = {
 	[EVMNetwork.METIS]: 15 * 1024,
 	[EVMNetwork.ASTAR]: 15 * 1024,
 };
+
+export const SendBulkMailTypes = {
```

Comment:
these constants are specific for a certain contract (MailerV9), I think it is reasonable to keep them in some MailerV9-related file, instead of a global one

Reply:
This constant with high probability will not change with further releases of Mailer contract.

Reply:
Agree, will put in Wrapper itself.